movie-app-code-review-main/src/app/components/header/header.ts

```diff
@@ -0,0 +1,39 @@
+import rsSchoolLogo from '@assets/rs_school.svg';
+import { BaseComponent } from '@components/base-component';
+import { a, div, h2, img } from '@components/tags';
+
+import styles from './header.module.scss';
+
+export const Header = () => {
+  // eslint-disable-next-line no-constant-condition
+  if (false)
```

Comment:
this condition will always be false, so the code inside will never be executed, this is an error